src/OpenLoco/Input/ShortcutManager.cpp

```diff
@@ -455,7 +484,7 @@ namespace OpenLoco::Input::ShortcutManager
         if (isEditorMode())
             return;
 
-        Ui::Windows::CompanyWindow::open(CompanyManager::getControllingId());
+        toggleWindow(WindowType::company, Windows::CompanyWindow::open, CompanyManager::getControllingId());
```

Comment:
Does this actually work. You will probably end up closing an unrelated companies window with this button. Say i had 3 company windows open. My understanding is that 3 presses of this shortcut would close the 3 windows and not open three players company window until the 4th press.